examples/hackernews/src/main.rs

```diff
@@ -0,0 +1,110 @@
+use cfg_if::cfg_if;
+use leptos::*;
+
+// boilerplate to run in different modes
+cfg_if! {
+    if #[cfg(feature = "ssr")] {
+        use actix_files::{Files, NamedFile};
+        use actix_web::*;
+        use futures::StreamExt;
+        use leptos_meta::*;
+        use leptos_router::*;
+        use leptos_hackernews*;
+    #[get("/static/style.css")]
+    async fn css() -> impl Responder {
+        NamedFile::open_async("./style.css").await
+    }
+
+    // match every path — our router will handle actual dispatch
+    #[get("{tail:.*}")]
+    async fn render_app(req: HttpRequest) -> impl Responder {
+        let path = req.path();
+
+        let query = req.query_string();
+        let path = if query.is_empty() {
+            "http://leptos".to_string() + path
+        } else {
+            "http://leptos".to_string() + path + "?" + query
+        };
+
+        let app = move |cx| {
+            let integration = ServerIntegration { path: path.clone() };
+            provide_context(cx, RouterIntegrationContext::new(integration));
+
+            view! { cx, <App/> }
+        };
+
+        let head = r#"<!DOCTYPE html>
+            <html lang="en">
+                <head>
+                    <meta charset="utf-8"/>
+                    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+                    <script type="module">import init, { main } from '/pkg/hackernews_client.js'; init().then(main);</script>"#;
```

Comment:
`/pkg/hackernews_client.js` here should probably be `/pkg/leptos_hackernews.js`

Reply:
Yep, missed that one. Corrected!